packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testGetIconIfEqualsThreshold.st

```diff
@@ -0,0 +1,14 @@
+tests
+testGetIconIfEqualsThreshold
+
+	| model messageTally actual expected |
+	
+	model := IPTTreeModelMock new. 
+	messageTally := MessageTallyMock new. 
+
+	messageTally tally: 50.
+	model totalTime: 100.
+	
+	expected := model runtimeIcon bits.
+	actual := (model getIcon: messageTally) bits.
+	self assert: expected equals: actual
```

Comment:
Please consider inserting a newline before the assert to visually separate the object construction from the actual test.